rasa_core/policies/fallback.py

```diff
@@ -87,7 +88,7 @@ def predict_action_probabilities(self,
         The fallback action is predicted if the NLU confidence is low
         or no other policy has a high-confidence prediction.
         """
-
+        self.core_threshold_set = False
```

Comment:
i like the implementation of the flag, but think the `core_threshold_set` name could be confusing (`core_threshold_set` makes me think more that the user overwrote the default threshold in their policy config). I think the action is something more similar to `should_nlu_fallback`, so maybe `should_core_fallback`?